plugins/ufm_log_analyzer_plugin/src/loganalyze/log_analyzer.py

```diff
@@ -337,10 +354,14 @@ def create_analyzer(parsed_args, full_extracted_logs_list,
         pdf_header = (
             f"Dump analysis for {os.path.basename(args.location)}, hours={args.hours}"
         )
-        fabric_info = ibdiagnet_analyzer.get_fabric_size() \
-                        if ibdiagnet_analyzer else "No Fabric Info found"
+        FABRIC_INFO = str(ibdiagnet_analyzer.get_fabric_size() \

```

Comment:
why are you using capital letters? is it a global variable that used somewhere else?

Reply:
No
Pylint made me!!!!

The error said that it is a constant var, and I just wanted to make him happy and leave me alone :)

I can bring it back to be lower and + add a Pylint rule to not look at it.
What do you think?

Reply:
weird. could you please write here the exact error you see from pylint?

Reply:
src/loganalyze/log_analyzer.py:357:8: C0103: Constant name "fabric_info" doesn't conform to UPPER_CASE naming style (invalid-name)
src/loganalyze/log_analyzer.py:360:8: C0103: Constant name "link_flapping" doesn't conform to UPPER_CASE naming style (invalid-name)
src/loganalyze/log_analyzer.py:363:8: C0103: Constant name "text" doesn't conform to UPPER_CASE naming style (invalid-name)